app/lib/activitypub/activity/create.rb

```diff
@@ -358,6 +358,8 @@ def visibility_from_audience
       :unlisted
     elsif equals_or_includes?(audience_to, @account.followers_url)
       :private
+    elsif !direct_message.nil? && !direct_message
```

Comment:
`direct_message == false` may be more readable

Reply:
I meant that the whole `!direct_message.nil? && !direct_message` could be replaced with `direct_message == false`!